forch/faucet_state_collector.py

```diff
@@ -1025,6 +1025,12 @@ def process_port_state(self, timestamp, name, port, state):
             port_table[PORT_STATE_TS] = datetime.fromtimestamp(timestamp).isoformat()
             port_table[PORT_STATE_COUNT] = port_table.setdefault(PORT_STATE_COUNT, 0) + 1
 
+            port_attr = self._get_port_attributes(name, port)
+            if port_attr and port_attr['type'] == 'access':
+                if self._placement_callback:
+                    devices_placement = DevicePlacement(switch=name, port=port, connected=False)
```

Comment:
Seems that `connected` is always False. Is this correct?

Reply:
Oh the check seems to have gotten deleted in one of the changes. Let me update

Reply:
Changed.